crates/shared/src/bad_token/token_owner_finder/blockscout.rs

```diff
@@ -53,8 +60,13 @@ impl BlockscoutTokenOwnerFinder {
             .append_pair("action", "getTokenHolders")
             .append_pair("contractaddress", &format!("{token:#x}"));
 
+        // Don't log the API key!
         tracing::debug!(%url, "Querying Blockscout API");
 
+        if let Some(api_key) = &self.api_key {
```

Comment:
Are we use the API key is appended to the query with the key `apiKey`?

Reply:
As the [API docs](https://docs.blockscout.com/for-users/my-account/api-keys#use-api-key) state.